tests/installation/bootloader_spvm.pm

```diff
@@ -0,0 +1,102 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Bootloader to setup boot process on powerVM
+# Maintainer: Stephan Kulow <coolo@suse.com>
+
+use base "installbasetest";
+use strict;
+
+use testapi;
+use lockapi;
+use bootloader_setup;
+use registration;
+
+sub get_into_net_boot {
+    assert_screen 'pvm-bootmenu';
+
+    # 5.   Select Boot Options
+    type_string "5\n";
+    assert_screen 'pvm-bootmenu-boot-order';
+
+    # 1.   Select Install/Boot Device
+    type_string "1\n";
+    assert_screen 'pvm-bootmenu-boot-device-type';
+
+    # 4.   Network
+    type_string "4\n";
+    assert_screen 'pvm-bootmenu-boot-network-service';
+
+    # 1.   BOOTP
+    type_string "1\n";
+    assert_screen 'pvm-bootmenu-boot-select-device';
+
+    # primary disk
+    type_string "1\n";
+    assert_screen 'pvm-bootmenu-boot-mode';
+
+    # 2.   Normal Mode Boot
+    type_string "2\n";
+    assert_screen 'pvm-bootmenu-boot-exit';
+
+    type_string "1\n";
+    assert_screen ["pvm-grub", "novalink-failed-first-boot"];
+}
+
+sub run {
+    my $lpar_id  = get_required_var('NOVALINK_LPAR_ID');
+    my $novalink = select_console 'novalink-ssh';
+
+    # detach possibly attached terminals - might be left over
+    type_string "rmvterm --id $lpar_id && echo 'DONE'\n";
+    assert_screen 'pvm-vterm-closed';
+
+    # power off the machine if it's still running - and don't give it a 2nd chance
+    type_string " pvmctl lpar power-off -i id=$lpar_id --hard\n";
+    assert_screen [qw(pvm-poweroff-successful pvm-poweroff-not-running)], 180;
+
+    # we assume the lpar is configured to boot normally
+    # pvmctl lpar update -i id=<LPARID> --set-fields LogicalPartition.bootmode=Normal
+    type_string " pvmctl lpar power-on -i id=$lpar_id --bootmode sms\n";
+    assert_screen [qw(pvm-poweron-successful)];
```

Comment:
I think `[qw(...)]` is not really useful for searching one tag.

Reply:
fixed